cpp/test/CMakeLists.txt

```diff
@@ -325,22 +316,18 @@ if(BUILD_TESTS)
     sparse/spgemmi.cu
     sparse/spmm.cu
     sparse/symmetrize.cu
+    sparse/preprocess_csr.cu
```

Comment:
I only needed to add these two files but it seems like the formatter wanted me to make the other changes?